<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connect_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Up (w)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Left (a)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Right (d)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Down (s)</string>
    </property>
   </widget>
   <widget class="QSlider" name="speed_slider">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>18</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>240</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BWebCam">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>WebCam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Batterie :</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IR Haut Gauche :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IR Haut Droit :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IR Bas Gauche :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down_button_2">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>510</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Down (down)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Right (right)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up_button_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Up (up)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button_2">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>480</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Left (left)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WEBCAM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>410</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ROBOT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
